src/EventListener/LogRevisionsListener.php

```diff
@@ -550,7 +554,7 @@ private function saveRevisionEntityData(EntityManagerInterface $em, ClassMetadat
                             $types[] = $targetClass->getTypeOfField($targetClass->getFieldForColumn($targetColumn));
                         }
                     }
-                } elseif (self::isManyToManyOwningSideMapping($assoc)) {
+                } elseif (!$this->config->areAssociationsDisabled() && self::isManyToManyOwningSideMapping($assoc)) {
```

Comment:
Why you added the check only here and you didn't skip the whole
```
foreach ($class->associationMappings
```
like everywhere else ?